src/rock/models/finances/models/__tests__/TransactionJobs.js

```diff
@@ -615,7 +617,7 @@ describe("findOrCreateTransaction", () => {
       CreatedByPersonAliasId: 10,
       ModifiedByPersonAliasId: 10,
       BatchId: 30,
-      SourceTypeValueId: NATIVE_SOURCE_TYPE,
+      SourceTypeValueId: WEB_SOURCE_TYPE,
```

Comment:
This test is trying to prove that you would get the type to switch to something else if you passed it a different platform type. We probably need to change this to test that you would get back iOS and Android types instead of just changing it to be web for everything.